src/background-plugin.ts

```diff
@@ -192,16 +182,45 @@ class BackgroundPlugin {
     return data;
   }
 
+  async getExports(name?: string): Promise<WebAssembly.ModuleExportDescriptor[]> {
+    return await this.#invoke('getExports', name ?? '0');
+  }
+
+  async getImports(name?: string): Promise<WebAssembly.ModuleImportDescriptor[]> {
+    return await this.#invoke('getImports', name ?? '0');
+  }
+
+  async getInstance(): Promise<WebAssembly.Instance> {
+    throw new Error('todo');
+  }
+
+  async close(): Promise<void> {
+    if (this.worker) {
+      this.worker.terminate();
+      this.worker = null as any;
+    }
+  }
```

Comment:
These functions moved up to put the `RingBufferWriter` closer to `#handleInvoke`.